src/status_im/chat/sign_up.cljs

```diff
@@ -189,4 +186,7 @@
   {:whisper-identity console-chat-id
    :name             (s/capitalize console-chat-id)
    :photo-path       console-chat-id
-   :dapp?            true})
+   :dapp?            true
+   ; todo remove/change dapp config fot console
+   :dapp-url         "http://localhost:8185/resources"
```

Comment:
It is not used at the moment, we discussed once that dapps will provide some url for js that will use our API to add command/requests